wasmcloud-test-util/src/provider_test.rs

```diff
@@ -397,14 +412,15 @@ pub async fn start_provider_test(
 /// This is like the `run_selected!` macro, except that it spawns
 /// a thread for running the test case, so it can handle panics
 /// (and failed assertions, which panic).
+/// Users of this macro must have a the testing interface package in scope by using `use wasmcloud_test_util::testing`;
```

Comment:
Generally macros should be using full paths and not requiring users to have paths imported. I'd greatly prefer that this macro uses the fully qualified paths instead. Same goes for the other macro as well

Reply:
Yes! All macros _do_ use full paths everywhere possible. Typically, "full path" means a crate name in Cargo.toml, but if you use smithy-bindgen, you aren't importing the testing crate - you created the namespace with 
```
mod testing { smithy_bindgen!( ...) }
```
so "full path" in this situation refers to the name of the module created.

Reply:
Ok, this comment is a bit misleading then. But I still think I am correct here. You are exporting the bindgen as `wasmcloud_test_util::testing` in the crate, so for now I think the stuff on line 451 should be `wasmcloud_test_util::testing::TestResult`. If people only pull in the testing stuff via interface, which I don't think is the primary use case

Reply:
no. TestResult is in wasmcloud_interface_testing, but that crate won't be available to clients. I'll update the comment with a code sample to clarify.

Reply:
Fixed it. Works with `wasmcloud_test_util::` prefix